static/js/jira-server-url.js

```diff
@@ -89,12 +124,17 @@ const verifyGitHubServerUrl = (gheServerURL, installationId) => {
 						}
 					);
 				} else {
-					console.error(`Failed to verify GHE server url. ${gheServerURL}`);
-					requestFailed();
-					// TODO - render error component
+					const { errorCode } = data;
+					const errorMessage = mapErrorCode(errorCode);
+					handleGheUrlRequestErrors(errorMessage);
 				}
+			},
+			error: function(err) {
+				const { errorCode } = JSON.parse(err.responseText);
+				const errorMessage = mapErrorCode(errorCode);
+				handleGheUrlRequestErrors(errorMessage);
```

Comment:
should probably have a single function to handle errors? code's pretty much the same.